libraries/ESP8266WiFi/src/lwip/mem_manager.h

```diff
@@ -69,9 +69,9 @@ static const unsigned short heapSTRUCT_SIZE	= ( sizeof( xBlockLink ) + portBYTE_
 
 void vApplicationMallocFailedHook( void ) ;//ICACHE_FLASH_ATTR;
 
-void *pvPortMalloc( size_t xWantedSize ) ;//ICACHE_FLASH_ATTR;
+void *pvPortMalloc( size_t xWantedSize, const char* file, int line ) ;//ICACHE_FLASH_ATTR;
```

Comment:
void *pvPortMalloc(size_t xWantedSize, const char* file, int line) __attribute__((malloc, alloc_size(1)));
is better for the compiler optimization